homeassistant/components/fortios/device_tracker.py

```diff
@@ -0,0 +1,99 @@
+"""
+Support to use FortiOS device like FortiGate as device tracker.
+
+This component is part of the device_tracker platform.
+"""
+import logging
+import voluptuous as vol
```

Comment:
Please add a blank line between standard library and 3rd party import groups.

Reply:
done